modules/internal/ChapelBase.chpl

```diff
@@ -1882,11 +1885,29 @@ module ChapelBase {
   pragma "no doc"
   inline proc _defaultOf(type t) where t: _sync_aux_t return _nullSyncVarAuxFields;
 
-  pragma "no doc"
+  /*pragma "no doc"
   inline proc _defaultOf(type t) where t: _ddata
-    return __primitive("cast", t, nil);
+    return __primitive("cast", t, nil);*/
```

Comment:
Let's clean this up before merging.
Or leave it commented out for now if you prefer.
If so, I'd prefer having /* and */ on their own lines, for cleaner revision history.